gpbft/api.go

```diff
@@ -94,6 +94,9 @@ type Network interface {
 	NetworkName() NetworkName
 	// Requests that the message is signed and broadcasted, it should also be delivered locally
 	RequestBroadcast(mb *MessageBuilder) error
+	// RequestRebroadcast requests that a message at given instance, round and phase
+	// previously broadcasted via RequestBroadcast be rebroadcasted.
+	RequestRebroadcast(instance, round uint64, phase Phase) error
```

Comment:
nit: maybe it's time to create some form of `gpbft.Instant` struct specifying an instance/round/phase?

Reply:
That's fine as long as `Instant` and `instance` are not confusing to the reader.

Reply:
I will do this in a separate PR if that's not a blocker, since i'd like to work it into Progress too.

Reply:
Yep, not critical. Just something that keeps coming up.